<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_R1">
      <property name="text">
       <string>Введите значение r1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_R1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_R2">
      <property name="text">
       <string>Введите значение r2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_R2">
      <property name="maximumSize">
       <size>
           <width>16777215</width>
           <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="seriesButton">
      <property name="text">
       <string>Последовательное соединение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="parallelButton">
      <property name="text">
       <string>Параллельное соединение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Result"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
